packages/blade/src/components/Accordion/AccordionButton.web.tsx

```diff
@@ -42,16 +52,17 @@ const AccordionButton = ({ index, icon: Icon, children }: AccordionButtonProps):
          */
         {...makeAccessible({ role: 'button' })}
         tabIndex={0}
-        // TODO: add logic
-        isExpanded={false}
+        isExpanded={expandedIndex === index}
+        // TODO: also handle keyboard
         onClick={onClick}
+        onKeyDown={onKeyDown}
       >
-        <BaseBox display="flex" flexDirection="row" alignItems="flex-start">
+        <BaseBox display="flex" flexDirection="row" alignItems="flex-start" marginRight="spacing.4">
           {_index}
           {_icon}
           <Heading size="small">{children}</Heading>
         </BaseBox>
-        <ChevronDownIcon color="currentColor" size="large" marginLeft="spacing.4" />
+        <CollapsibleChevronIcon color="currentColor" size="large" />
```

Comment:
Right, size needs to be one size above the button / link size

Reply:
Ah wait, I got confused, this will always be size large for accordion, accordion doesn't has any size related props. For link the size is automatically controlled by the `Link` component.

Reply:
no my question was the size of the icon should be the same size of the link component right?

Reply:
This is being rendered inside the AccordionButton, here it should always be large. For the CollapsibleLink the size is automatically controlled by the Link component.